docs/quota_management.md

```diff
@@ -52,7 +52,7 @@ Users can also implement their own identity provider by inheriting the `org.apac
 ## QuotaManager
 
 `QuotaManager` supports to check whether quota is available and manage quota configurations for `Master`.
-`QuotaManager` uses the [dynamic config service](developers/configuration#dynamic-configuration) to store quota settings.
+`QuotaManager` uses the [dynamic config service](https://celeborn.apache.org/docs/latest/developers/configuration/#dynamic-configuration) to store quota settings.
```

Comment:
`../developers/configuration#dynamic-configuration`?

Reply:
../developers/configuration#dynamic-configuration is not correct, it should be ./developers/configuration#dynamic-configuration, I'm not sure if it will display correctly on the web page.

Reply:
@leixm, why is `../developers/configuration#dynamic-configuration` not correct. You could refer to the link in `client.md`.

Reply:
> @leixm, why is `../developers/configuration#dynamic-configuration` not correct. You could refer to the link in `client.md`.

You are right.